src/test/resources/fftogff3_rules/locus_tag.gff3

```diff
@@ -2,4 +2,4 @@
 ##sequence-region TEST01.1 1 822
 TEST01.1	.	ncRNA_gene	1	822	.	+	.	ID=ncRNA_gene_matK;gene=matK;locus_tag=test;
 TEST01.1	.	mRNA	1	822	.	+	.	ID=mRNA_matK;Parent=ncRNA_gene_matK;locus_tag=test;
-TEST01.1	.	mRNA	1000	8220	.	+	.	ID=mRNA_matK_1;Parent=ncRNA_gene_matK;locus_tag=different-locus;
\ No newline at end of file
+TEST01.1	.	mRNA	1000	8220	.	+	.	ID=mRNA_matK_1;Parent=ncRNA_gene_matK;locus_tag=test;
```

Comment:
This test was specifically to retain different locus_tag. We. shouldn't override the existing locus_tag but add one when it's not present.